src/dsymbol.d

```diff
@@ -1207,6 +1207,9 @@ private:
     Dsymbols* importedScopes;
     PROTKIND* prots;            // array of PROTKIND, one for each import
 
+    import std.bitmanip : BitArray;
+    BitArray accessiblePackages;// whitelist of accessible (imported) packages
```

Comment:
The dependency goes from 2.067 to dmd master, not from the last release.  Since we almost certainly want to be compatible with whatever LDC and GDC releases are most recent, this gap doesn't look like it will shrink soon.

We don't have a choice when it comes to druntime.  We do with phobos.

Can we put this discussion off until dconf?  It doesn't look like this PR couldn't be done without it.

I suspect use of phobos, especially in the C++-boundary API, is going to make my work on automatic generation of headers much more difficult.